packages/bus-merge-binding-factory/lib/ActorMergeBindingFactory.ts

```diff
@@ -0,0 +1,45 @@
+import type { IAction, IActorArgs, IActorOutput, IActorTest, Mediate } from '@comunica/core';
+import { Actor } from '@comunica/core';
+import type { IActionContext } from '@comunica/types';
+
+/**
+ * A comunica actor for the creation of merge handlers for binding context keys.
+ *
+ * Actor types:
+ * * Input:  IActionMergeBindingFactory:      TODO: fill in.
+ * * Test:   <none>
+ * * Output: IActorMergeBindingFactoryOutput: TODO: fill in.
+ *
+ * @see IActionMergeBindingFactory
+ * @see IActorMergeBindingFactoryOutput
+ */
+
+export abstract class ActorMergeBindingFactory extends Actor<IActionMergeBindingFactory,
+IActorTest,
+IActorMergeBindingFactoryOutput> {
+  /**
+  * @param args - @defaultNested {<default_bus> a <cc:components/Bus.jsonld#Bus>} bus
+  */
+  public constructor(args: IActorMergeBindingFactoryArgs) {
+    super(args);
+  }
+}
+
+export interface IActionMergeBindingFactory extends IAction {
+  context: IActionContext;
+}
+
+export interface IActorMergeBindingFactoryOutput extends IActorOutput {
+  mergeHandlers: Record<string, IMergeHandler<any>>;
+}
+
+export type IActorMergeBindingFactoryArgs = IActorArgs<
+IActionMergeBindingFactory, IActorTest, IActorMergeBindingFactoryOutput>;
+
+export type MediatorMergeBindingFactory = Mediate<
+IActionMergeBindingFactory, IActorMergeBindingFactoryOutput>;
+
+export interface IMergeHandler<V> {
```

Comment:
Suggestion for a more specific name: `IBindingsContextMergeHandler`.

Also add some jsdoc here explaining what it is and does.
Make sure to add in jsdoc that `V` is the type corresponding to the context key.